libindy/src/commands/anoncreds/tails.rs

```diff
@@ -80,7 +80,7 @@ pub fn store_tails_from_generator(service: Rc<BlobStorageService>,
     let version = vec![0u8, TAILS_BLOB_TAG_SZ];
     service.append(blob_handle, version.as_slice())?;
 
-    while let Some(tail) = rtg.next()? {
+    while let Some(tail) = rtg.try_next()? {
```

Comment:
compare differences between **next()** and **try_next()**

Reply:
The functionality appears to do the same things. However, this new function looks like it's generating a new Tails object each time and returning the new object rather than iterating on the same object. I was not able to find the impl of the **next()** trait in the std library to verify they do it differently. My only concern with this would be that it's not as memory efficient, which given the size of the Tails object may be a problem. Was there a reason for switching this?

Reply:
No problems here. This is not referring to the trait **next()** which is performed on an iterator, but rather to the **next()** function which can be found in Indy_crypto here: https://github.com/hyperledger/indy-crypto/blob/ca98135687a0674722f94836a2359ac5fd629b54/libindy-crypto/src/cl/mod.rs#L504